src/scipp/visualizations/__init__.py

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Can you make this singular, `visualization`? I think having plural on module names is not so common?